### ISSUE TITLE
el.cycle needs a frequency value

### ISSUE THREAD

**Issue by nikola-n6c** [nick-thompson/elementary#9]
hey, 

i was trying out the example from https://docs.elementary.audio/guides/understanding_keys

```js
const core = require('elementary-core');
const el = require('@nick-thompson/elementary');

core.on('load', function() {
	let x = el.cycle({key: 'test', value: 440});
	let y = el.cycle({key: 'test', value: 441});
	

	core.render(x);

	core.render(y);
});
```
but getting:
```
$ el main.js
el.cycle needs a frequency value! Received: undefined
el.cycle needs a frequency value! Received: undefined
```

i'm running `0.2.2`, but I figured this part of elementary is not open sourced. what's the next best way to dig into issues like these? 

thanks!

**Comment by nick-thompson** [comment]
Yikes, that's just an incorrect example! It should be written:

```
let x = el.cycle(el.const({key: 'test', value: 440}));
let y = el.cycle(el.const({key: 'test', value: 441}));

core.render(x);
core.render(y);
```

Thanks for the heads up, I'll fix the docs there

**Comment by nick-thompson** [comment]
Fixed in 29c11ca7b27e453fbe6877f3737ded9b2d541d69. Thank you!
